.gitignore

```diff
@@ -19,6 +19,9 @@ admin.conf
 # templated ci values
 charts/redpanda/ci/21-eks-tiered-storage-with-creds-values.yaml
 
+# Dep Chart Lock file
+Chart.lock
```

Comment:
So we ignore the file we can commit but then generate? Is this the intent?

Reply:
I was testing if `git add --all` will omit redpanda `Chart.lock`, but turns out once you commit particular file git will track it regardless of the `.gitignore`. I can remove this, but do we want Chart.lock from other charts beside Redpanda?